File: java/FibonacciSeries.java
import java.util.Scanner;
public class FibonacciSeries{
    public static void main(String args[]){
        int n;
        Scanner sc = new Scanner(System.in);
        System.out.println("Enter the value of n.");
        n= sc.nextInt();
        int a = 0;
        int b = 1;
        System.out.println("The fibonacci series till term number "+n+" is:");
        for(int i = 1; i <=(n+1)/2; i++){
            System.out.print(a+" ");
            if(n%2!=0 && i==(n+1)/2){
                break;
            }
            else{
                System.out.print(b+" ");
                a+=b;
                b+=a;
            }
            
        }
        sc.close();

    }
}

File: python/Fibonacci_Series.py
def FibonacciSeriesByTerms():
    nums = int(input("Enter number of terms: "))
    a = 0
    b = 1
    i = 0
    for i in range(0,nums):
        c = b
        b = a+b
        a = c
        print(a)

def FibonacciSeriesByMaxValue():
    nums = int(input("Enter the max value: "))
    a = 0
    b = 1
    while(b<=nums):
        c = b
        b = a+b
        a = c
        print(a)

print("\n","-"*20,"Fibonacci Series","-"*20)
print("\n1.By no. of terms")
print("\n2.By max value")
a = int(input("\nEnter your choice[1 or 2]: "))
if(a == 1):FibonacciSeriesByTerms()
else:
    if(a == 2):FibonacciSeriesByMaxValue()
    else:print("\nInvalid Input")